### ISSUE TITLE
main/java and main/resources do not appear in classpath of integrationTest for version 2.0.0

### ISSUE THREAD

**Issue by dje1990** [unbroken-dome/gradle-testsets-plugin#55]
Using version 2.0.0 of the plugin, when running a test set (e.g. integrationTest) the runtime classpath does not contain the output of main/java and main/resources.

Using gradle 4.10.2 and jdk1.8.0_181, using the following source files, when running integrationTest the class Foobar cannot be found in the runtime classpath.

The example below works correctly with plugin version 1.5.1 but not with plugin version 2.0.0

build.gradle:

```
plugins {
//    id "org.unbroken-dome.test-sets" version "1.5.1" // works with 1.5.1
    id "org.unbroken-dome.test-sets" version "2.0.0" // does not work with 2.0.0
}

group 'dje1990'
version 'HEAD-SNAPSHOT'

apply plugin: "java"
apply plugin: "idea"

sourceCompatibility = 1.8

repositories {
    mavenCentral()
}

dependencies {
    testCompile group: "org.junit.jupiter", name:"junit-jupiter-engine", version:"5.3.1"
}

test {
    useJUnitPlatform()
}

testSets {
    integrationTest
}

integrationTest {
    useJUnitPlatform()
}

sourceSets {
    integrationTest {
        java {
            srcDir "src/integrationTest/java"
        }
        resources {
            srcDir "src/integrationTest/resources"
        }
        compileClasspath += sourceSets.main.runtimeClasspath
    }
}

idea {
    module {
        testSourceDirs += project.sourceSets.integrationTest.java.srcDirs
        testSourceDirs += project.sourceSets.integrationTest.resources.srcDirs
    }
}

check.dependsOn integrationTest
integrationTest.mustRunAfter test
```

main/java/.../Foobar.java
```
package dje1990.testTestSets;

public class Foobar {
}
```

test/java/.../FoobarTest.java
```
package dje1990.testTestSets;

import org.junit.jupiter.api.Test;

public class FoobarTest {

    @Test
    public void test() {
        new Foobar();
    }
}
```

integrationTest/java/.../FoobarIntegrationTest.java
```
package dje1990.testTestSets;

import org.junit.jupiter.api.Test;

public class FoobarIntegrationTest {

    @Test
    public void test() {
        new Foobar();
    }
}
```

gradle output:
```
Testing started at 20:24 ...
20:24:47: Executing task 'integrationTest'...

> Task :compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :compileIntegrationTestJava UP-TO-DATE
> Task :processIntegrationTestResources NO-SOURCE
> Task :integrationTestClasses UP-TO-DATE
> Task :integrationTest FAILED
dje1990.testTestSets.FoobarIntegrationTest > test() FAILED
    java.lang.NoClassDefFoundError at FoobarIntegrationTest.java:9
        Caused by: java.lang.ClassNotFoundException at FoobarIntegrationTest.java:9
1 test completed, 1 failed
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':integrationTest'.
> There were failing tests. See the report at: file:///C:/Users/david/IdeaProjects/testTestSets/build/reports/tests/integrationTest/index.html
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org
BUILD FAILED in 1s
3 actionable tasks: 1 executed, 2 up-to-date

dje1990/testTestSets/Foobar
java.lang.NoClassDefFoundError: dje1990/testTestSets/Foobar
	at dje1990.testTestSets.FoobarIntegrationTest.test(FoobarIntegrationTest.java:9)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:515)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:115)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$6(TestMethodTestDescriptor.java:171)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:72)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:167)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:114)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:59)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:105)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:72)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:71)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:110)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:72)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:71)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:110)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:72)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:71)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:170)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:154)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:90)
	at org.gradle.api.internal.tasks.testing.junitplatform.JUnitPlatformTestClassProcessor$CollectAllTestClassesExecutor.processAllTestClasses(JUnitPlatformTestClassProcessor.java:92)
	at org.gradle.api.internal.tasks.testing.junitplatform.JUnitPlatformTestClassProcessor$CollectAllTestClassesExecutor.access$100(JUnitPlatformTestClassProcessor.java:77)
	at org.gradle.api.internal.tasks.testing.junitplatform.JUnitPlatformTestClassProcessor.stop(JUnitPlatformTestClassProcessor.java:73)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.stop(SuiteTestClassProcessor.java:61)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy2.stop(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.stop(TestWorker.java:131)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:155)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:137)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.ClassNotFoundException: dje1990.testTestSets.Foobar
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:582)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	... 63 more

There were failing tests. See the report at: file:///C:/Users/david/IdeaProjects/testTestSets/build/reports/tests/integrationTest/index.html
20:24:48: Task execution finished 'integrationTest'.

```

**Comment by dje1990** [comment]
The same result occurs when using JUnit 4.12 instead of JUnit 5.3.1

**Comment by tkrullmann** [comment]
That was quick, I haven't even used the new version myself (obviously ;-) )

Blatant oversight... I modified the classpath of the Test task with `=` instead of `+=`, effectively replacing the classpath that was already there.

Thanks for the detailed example!

**Comment by Zar86** [comment]
id 'org.unbroken-dome.test-sets' version '2.0.2' //does not work

Testing started at 15:43 ...
15:43:34: Executing task 'integrationTest'...

Parallel execution is an incubating feature.
> Task :compileJava UP-TO-DATE
> Task :processResources UP-TO-DATE
> Task :eclipselinkWeaveClasses
> Task :classes
> Task :compileIntegrationTestJava FAILED
....
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':compileIntegrationTestJava'.
> Compilation failed; see the compiler error output for details.


id 'org.unbroken-dome.test-sets' version '1.5.2' //work

**Comment by dje1990** [comment]
Hi @Zar86,
The original issue in this thread was that the compiled classes were not available at integration test runtime.
This looks like the exception happened at the compile time for the integration tests. 
Do you have more output describing the exception to better understand the issue? 
Just looking at the above, your issue looks like a separate issue than the original issue in this thread. 
Thanks

**Comment by tkrullmann** [comment]
@Zar86, can you post a more detailed example / include the error message from the compiler?

**Comment by Zar86** [comment]
Sorry for the delayed response.

id 'org.unbroken-dome.test-sets' version '2.0.2'
```
Testing started at 8:55 ...
8:55:28: Executing task 'integrationTest'...

Parallel execution is an incubating feature.
> Task :compileJava
Note: Creating static metadata factory ...
Note: Optional file was not found: META-INF/orm.xml continuing with generation.
Note: Optional file was not found: META-INF/eclipselink-orm.xml continuing with generation.
Note: Optional file was not found: META-INF/orm.xml continuing with generation.
Note: Optional file was not found: META-INF/eclipselink-orm.xml continuing with generation.
Note: /Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/main/java/ru/citc/fcb/mfc/internal/DomainEventsWebSocketPublisher.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
> Task :processResources
> Task :eclipselinkWeaveClasses
> Task :classes
> Task :compileIntegrationTestJava FAILED
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/SyncServiceTest.java:16: error: cannot find symbol
import ru.citc.fcb.mfc.internal.domain.ContactTreeNode;
                                      ^
  symbol:   class ContactTreeNode
  location: package ru.citc.fcb.mfc.internal.domain
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/SyncServiceTest.java:17: error: cannot find symbol
import ru.citc.fcb.mfc.internal.domain.Debt;
                                      ^
  symbol:   class Debt
  location: package ru.citc.fcb.mfc.internal.domain
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/SyncServiceTest.java:18: error: cannot find symbol
import ru.citc.fcb.mfc.internal.domain.Debtor;
                                      ^
  symbol:   class Debtor
  location: package ru.citc.fcb.mfc.internal.domain
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/SyncServiceTest.java:19: error: cannot find symbol
import ru.citc.fcb.mfc.internal.domain.Device;
                                      ^
  symbol:   class Device
  location: package ru.citc.fcb.mfc.internal.domain
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/SyncServiceTest.java:32: error: cannot find symbol
    SyncService service;
    ^
  symbol:   class SyncService
  location: class SyncServiceTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:35: error: cannot find symbol
    private Repository repository;
            ^
  symbol:   class Repository
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/SyncServiceTest.java:47: error: cannot find symbol
        final Device device = new Device("test", "deviceId", "p");
              ^
  symbol:   class Device
  location: class SyncServiceTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/SyncServiceTest.java:47: error: cannot find symbol
        final Device device = new Device("test", "deviceId", "p");
                                  ^
  symbol:   class Device
  location: class SyncServiceTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/SyncServiceTest.java:56: error: cannot find symbol
        final Debt debt = entityManager.find(Debt.class, 1L);
              ^
  symbol:   class Debt
  location: class SyncServiceTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/SyncServiceTest.java:56: error: cannot find symbol
        final Debt debt = entityManager.find(Debt.class, 1L);
                                             ^
  symbol:   class Debt
  location: class SyncServiceTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/SyncServiceTest.java:59: error: cannot find symbol
        final Debtor debtor = entityManager.find(Debtor.class, 1L);
              ^
  symbol:   class Debtor
  location: class SyncServiceTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/SyncServiceTest.java:59: error: cannot find symbol
        final Debtor debtor = entityManager.find(Debtor.class, 1L);
                                                 ^
  symbol:   class Debtor
  location: class SyncServiceTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/SyncServiceTest.java:62: error: cannot find symbol
        final ContactTreeNode node = entityManager.find(ContactTreeNode.class, 4L);
              ^
  symbol:   class ContactTreeNode
  location: class SyncServiceTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/SyncServiceTest.java:62: error: cannot find symbol
        final ContactTreeNode node = entityManager.find(ContactTreeNode.class, 4L);
                                                        ^
  symbol:   class ContactTreeNode
  location: class SyncServiceTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:52: error: cannot find symbol
        final Debtor debtor = repository.createDebtor(1000L, "Test", false);
              ^
  symbol:   class Debtor
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:54: error: cannot find symbol
        final Device device = new Device("l", "deviceId", "p");
              ^
  symbol:   class Device
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:54: error: cannot find symbol
        final Device device = new Device("l", "deviceId", "p");
                                  ^
  symbol:   class Device
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:57: error: cannot find symbol
        final CollectorStatistic statistic = repository.createCollectorStatistic(device.getLogin(), "sac", BigDecimal.ONE, BigDecimal.TEN, 1, 1);
              ^
  symbol:   class CollectorStatistic
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:59: error: cannot find symbol
        final FilterType filterType = repository.createFilterType(1000, "f", true);
              ^
  symbol:   class FilterType
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:61: error: cannot find symbol
        final Debt newDebt = repository.createDebt(1000L,
              ^
  symbol:   class Debt
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:63: error: cannot find symbol
                DebtType.DCA,
                ^
  symbol:   variable DebtType
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:81: error: cannot find symbol
        final AddressType addressType = repository.createAddressType(1000, "dsada", true);
              ^
  symbol:   class AddressType
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:83: error: cannot find symbol
        final Address address = debtor.addAddress(1000L, addressType, "fcsadfasdasda", "dasdasdas", BigDecimal.valueOf(20.2), BigDecimal.valueOf(2222.3), true, false);
              ^
  symbol:   class Address
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:85: error: cannot find symbol
        newDebt.addAddressJob(1000L, Instant.now(), LocalDate.now(), JobContactType.AR_VYEZDPOADRESU, JobStatus.CREATED, "fsdfsdfsdfsd", false, address);
                                                                     ^
  symbol:   variable JobContactType
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:85: error: cannot find symbol
        newDebt.addAddressJob(1000L, Instant.now(), LocalDate.now(), JobContactType.AR_VYEZDPOADRESU, JobStatus.CREATED, "fsdfsdfsdfsd", false, address);
                                                                                                      ^
  symbol:   variable JobStatus
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:87: error: cannot find symbol
        final ContactTreeNode node = repository.createContactTreeNode(1000L, TreeContactType.AR_VYEZDPOADRESU, "name1", 0, "123355", false);
              ^
  symbol:   class ContactTreeNode
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:87: error: cannot find symbol
        final ContactTreeNode node = repository.createContactTreeNode(1000L, TreeContactType.AR_VYEZDPOADRESU, "name1", 0, "123355", false);
                                                                             ^
  symbol:   variable TreeContactType
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:89: error: cannot find symbol
        final FormFieldContactTree fieldContactTree = node.addFormFieldContactTree(1000L, "name1", "key", FieldType.STRING, 0, false);
              ^
  symbol:   class FormFieldContactTree
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:89: error: cannot find symbol
        final FormFieldContactTree fieldContactTree = node.addFormFieldContactTree(1000L, "name1", "key", FieldType.STRING, 0, false);
                                                                                                          ^
  symbol:   variable FieldType
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:94: error: cannot find symbol
        final Debt debt1 = entityManager.find(Debt.class, 1000L);
              ^
  symbol:   class Debt
  location: class RepositoryTest
/Users/azaretskiy/projects/citc/fcb/mfc/mfc-backend/src/integrationTest/java/ru/citc/fcb/mfc/internal/domain/RepositoryTest.java:94: error: cannot find symbol
        final Debt debt1 = entityManager.find(Debt.class, 1000L);
                                              ^
  symbol:   class Debt
  location: class RepositoryTest
31 errors
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':compileIntegrationTestJava'.
> Compilation failed; see the compiler error output for details.
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org
BUILD FAILED in 8s
```

**Comment by dje1990** [comment]
Hi @Zar86 

Do you have something like the below in your `build.gradle` where the compiled output of main is being made available to the integration test compile time?
```
sourceSets {
    integrationTest {
        java {
            srcDir "src/integrationTest/java"
        }
        resources {
            srcDir "src/integrationTest/resources"
        }
        compileClasspath += sourceSets.main.runtimeClasspath
    }
}
```

**Comment by Zar86** [comment]
After adding
```
sourceSets {
    integrationTest {
        compileClasspath += sourceSets.main.runtimeClasspath
    }
}
```
 to the build.gradle integration test was performed.
But in version 1.5.2, the test was run without adding this code. This is so conceived, and now in version 2.0.2 I need to always add this code?

**Comment by tkrullmann** [comment]
No, it should be done by the plugin automatically. I'll reopen the issue.

**Comment by Zar86** [comment]
Thanks

**Comment by tkrullmann** [comment]
Please try again with version 2.0.3

**Comment by Zar86** [comment]
Thanks for the prompt decision of a problem!

**Comment by tkrullmann** [comment]
This should be solved by #66 and the accompanying PR
